### ISSUE TITLE
NoWhitespaceAfter: add support for method reference operator

### ISSUE THREAD

**Issue by rnveach** [checkstyle/checkstyle#3950]
"::" is separator, NOT a operator - https://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-3.11

$ cat MethodRef.java

``` java
import java.util.Objects;
import java.util.stream.Stream;

public class MethodRef
{
    {
        Stream.of().filter(Objects :: nonNull); // 1 VIOLATIONS -- want no spaces around '::' operator
    }
}
```

$ cat config.xml

``` xml
<!DOCTYPE module PUBLIC
  "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
  "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="NoWhitespaceAfterCheck">
            <message key="ws.preceded" value="Whitespace after ''{0}'' should be removed" />
            <property name="tokens" value="METHOD_REF" />
        </module>
    </module>
</module>
```

/var/tmp $ java -jar checkstyle-7.0-all.jar -c config.xml MethodRef.java
C:\Users\U0076777>java -jar checkstyle-7.0-all.jar -c config.xml MethodRef.java
com.puppycrawl.tools.checkstyle.api.CheckstyleException: cannot initialize module TreeWalker - Token "METHOD_REF" was
not found in Acceptable tokens list in check com.puppycrawl.tools.checkstyle.checks.whitespace.NoWhitespaceAfterCheck
        at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:407)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:138)
        at com.puppycrawl.tools.checkstyle.Main.runCheckstyle(Main.java:385)
        at com.puppycrawl.tools.checkstyle.Main.runCli(Main.java:323)
        at com.puppycrawl.tools.checkstyle.Main.main(Main.java:161)
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Token "METHOD_REF" was not found in Acceptable tokens list in check com.puppycrawl.tools.checkstyle.checks.whitespace.NoWhitespaceAfterCheck
        at com.puppycrawl.tools.checkstyle.TreeWalker.registerCheck(TreeWalker.java:226)
        at com.puppycrawl.tools.checkstyle.TreeWalker.setupChild(TreeWalker.java:163)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:138)
        at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:402)
        ... 4 more
Checkstyle ends with 1 errors.

---

Want to enforce no whitespace around the new '::' operator.
##

**Comment by sagar-shah94** [comment]
I am on it

**Comment by rnveach** [comment]
@sagar-shah94 As I stated before, I think you should find harder Issues to work on and let new people take the easy issues.
We are waiting for fixes on atleast 4 of your PRs.

**Comment by sagar-shah94** [comment]
@rnveach i will start working on issue labeled medium.

**Comment by rnveach** [comment]
Pre-assigned to Suraj Kumar

**Comment by surajumang** [comment]
I am on it

**Comment by romani** [comment]
fix is merged, new token was added to getAcceptableTokens
